ReminderApp-OBJC/ReminderApp-OBJC/Classes/Controller/CDReminderTableViewController.m

```diff
@@ -1,35 +1,38 @@
 //
-//  ViewController.m
+//  CDReminderTableViewController.m
 //  ReminderApp-OBJC
 //
-//  Created by Catalin David on 01/07/16.
+//  Created by Catalin David on 11/07/16.
 //  Copyright © 2016 Catalin David. All rights reserved.
 //
 
 #import "CDReminderTableViewController.h"
+#import "CDAddReminderCell.h"
+#import "CDReminderCell.h"
 #import "AppDelegate.h"
 #import "CDReminder.h"
 
-#pragma mark - Class extension
+@interface CDReminderTableViewController () <UITextViewDelegate, NSFetchedResultsControllerDelegate>
 
-@interface CDReminderTableViewController () <NSFetchedResultsControllerDelegate>
-
-@property (nonatomic, weak) IBOutlet UIBarButtonItem *editButton;
+@property (nonatomic, assign) NSInteger count;
 @property (nonatomic, strong) NSFetchedResultsController *fetchedResultsController;
 @property (nonatomic, strong) NSManagedObjectContext *managedContext;
 
 @end
 
-#pragma mark - Class implementation
-
 @implementation CDReminderTableViewController
 
 - (void)viewDidLoad {
 	
 	[super viewDidLoad];
 	
-	self.editButton.title = @"Edit";
-	self.title = @"Reminder";
+	self.count = 1;
+	
+	self.tableView.rowHeight = UITableViewAutomaticDimension;
+	self.tableView.estimatedRowHeight = 100;
+	
+	[[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(keyboardWillShow:) name:UIKeyboardWillShowNotification object:nil];
```

Comment:
You must remove the observer in the dealoc method. Implement the dealoc method and remove the observers